lib/orbit/lib/diffs.js

```diff
@@ -91,6 +91,10 @@ var diffs = function(a, b, options) {
               bi++;
             }
           }
+        } else if (a instanceof Date) {
+          if (a.getTime() === b.getTime()) return;
```

Comment:
There's a check at https://github.com/orbitjs/orbit.js/blob/647d88654ccb9cbe41e9ace852b8b6692d7dcc4b/lib/orbit/lib/diffs.js#L46 to make sure the type of a and b match. If they're different it'll fall through to https://github.com/orbitjs/orbit.js/blob/647d88654ccb9cbe41e9ace852b8b6692d7dcc4b/lib/orbit/lib/diffs.js#L130 which just does a replace operation with a clone of b.